### PR TITLE
IaC Template for Spanner CDC Load to BigQuery for analytical use-cases

### PR DESCRIPTION
**Pull request by SMaktub**
Adding example code for automated pipeline for spanner change stream to BigQuery via Dataflow. 
This terraform code helps to deploy an end-to-end pipeline for creating spanner resources, create a change stream, create a bigquery dataset and a dataflow flex job to publish changes into bigquery.

### CODE REVIEW

**Comment by CYarros10** [comment]
A setup script / guide to deploy and run the solution for someone with no experience on your tool would be helpful for adoption

File: examples/spanner-changestreams-bigquery/variables.tf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "terraform_service_account" {
+  description = "Terraform Service Account to be impersonated."
```
Comment (shakeebshams) [inline examples/spanner-changestreams-bigquery/variables.tf:18]: please ensure standard tf spacing

**Review by shakeebshams — CHANGES_REQUESTED** [review]

File: examples/spanner-changestreams-bigquery/variables.tf (hunk 1 above)
Comment (SMaktub) [reply examples/spanner-changestreams-bigquery/variables.tf:18]: Checked the formatting and corrected as per best practices! Thanks for pointing this out.

**Comment by SMaktub** [comment]
The CLA checks which passed previously https://github.com/GoogleCloudPlatform/professional-services/pull/890/checks is failing for subsequent commits which required merged

**Review by shakeebshams — APPROVED** [review]

**Comment by SMaktub** [comment]
@agold-rh Could you please help merge this PR.

**Review by agold-rh — APPROVED** [review]

**Pull request merged**